docs/content/docs/dev/table/tuning.md

```diff
@@ -266,5 +266,23 @@ GROUP BY day
 Flink SQL optimizer can recognize the different filter arguments on the same distinct key. For example, in the above example, all the three COUNT DISTINCT are on `user_id` column.
 Then Flink can use just one shared state instance instead of three state instances to reduce state access and state size. In some workloads, this can get significant performance improvements.
 
+## MiniBatch Join
```

Comment:
Please also update this sentence, which should contain the regular join: https://github.com/apache/flink/pull/24240/files#diff-8d75f38adf6c9f25a579a8fd856b1946386d4aae2ecdc8bbdb2c2d548ac2a590R31

```In this page, we will introduce some useful optimization options and the internals of streaming aggregation, regular join which will bring great improvement in some cases.```